### ISSUE TITLE
Include support for React navigation v6

### ISSUE THREAD

**Issue by mdmusaibali** [torgeadelin/react-native-animated-nav-tab-bar#122]

**Comment by malinthann** [comment]
where?

**Comment by mdmusaibali** [comment]
Check package.json.
Currently this package uses react navigation v5.
This is causing dependency incompatibility if the project that we are making uses react navigation v6.

**Comment by malinthann** [comment]
Do you know how to fix it, or what react-native-animated-tab-bar that you prefer?

**Comment by aaron5670** [comment]
I'm also interested in an solution 😄

**Comment by tschleuss** [comment]
I managed to get it working on a personal project.. I'm not using the package, decided to copy the sources over to my project, and the only error was within `TabBarElement.tsx`, need to fix the usage of `Descriptor` type on L:25. 

I changed it to `Descriptor<any, any, any>` just for a quick testing, and everything is working.. 
I'll work on a proper type solution later.

Maybe I can face other problems trying to integrate it deeper with v6 navigators, but for now, just using it alone, it's working.
